src/common/errors.ts

```diff
@@ -0,0 +1,3 @@
+export interface ErrorWithStatus extends Error {
+  status?: number;
+}
```

Comment:
ErrorWithStatus is strange, rename to HttpError

Reply:
done